services/QuillLMS/app/models/feedback_history_rating.rb

```diff
@@ -0,0 +1,21 @@
+# == Schema Information
+#
+# Table name: feedback_history_ratings
+#
+#  id                  :integer          not null, primary key
+#  rating              :string           not null
+#  created_at          :datetime         not null
+#  updated_at          :datetime         not null
+#  feedback_history_id :integer          not null
+#  user_id             :integer          not null
+#
+# Foreign Keys
+#
+#  fk_rails_...  (user_id => users.id)
+#
+class FeedbackHistoryRating < ActiveRecord::Base
+    RATINGS = %w(strong weak unrated)
```

Comment:
Thank you for explaining your reasoning, but I prefer a non-boolean here because
-  it is more extensible. If we want a new rating category in the future, we won't have to add/drop a column
- it allows us to add a not-null constraint on the column. Kind of a minor thing, but I like not having to check for nils

Reply:
Okay. I still don't think we need `unrated`, because the lack of a record is enough to tell us that a given user has not rated a given piece of feedback. We're going to end up relying on the lack of record sometimes anyway, because it isn't really practical to autogenerate an `unrated` record for every matrix of user and feedback history as they're generated.

Reply:
Just to chime in here, I don't think I have super strong feelings, but it may be worth pointing out that we already use the `true/false/null` pattern in the CMS for `optimal`.  It's either `true` or `false`, indicating a value set by a human, or `null` when a human hasn't set a value.  It may be worth maintaining that pattern just because we're familiar with it.

Reply:
Actually, now that I look again, this may be a little misguided.  `unrated` is a value that can't be set because the presence of a row in this table (connecting a `user` and a `feedback_history`) implies that a rating has been made.  Lack of a row is the `null` or `unrated` value by implication.  So this is really just `strong`/`weak`.  Again, I'd probably lean toward a boolean value to mirror `optimal` in CMS `Response`, but we could still not-NULL it since existence implies a decision.